### PR TITLE
Solution

### PR DESCRIPTION
**Pull request by goldenuni**

### CODE REVIEW

**Review by ArsenPidhoretskyi — CHANGES_REQUESTED** [review]

File: app/main.py [hunk 1]
```diff
@@ -1 +1,9 @@
-#  write your code here
+from app.animal import Animal, Cat, Dog
+
+
+def feed_animals(ls_animals: list[Animal | Cat | Dog]) -> int:
```
[the remark this answers is not in the document]
Comment (goldenuni) [reply app/main.py:4]: @ArsenPidhoretskyi , yes but flake8 is arguing about that, cause Cat and Dog are not used

[the remark this answers is not in the document]
Comment (ArsenPidhoretskyi) [reply app/main.py:4]: Can you peen a screenshot?
Maybe because no need to import them here?

File: app/main.py [hunk 2]
```diff
@@ -1 +1,5 @@
...
```
Comment (ArsenPidhoretskyi) [inline app/main.py:4]: also rename ls_animals to animals

File: app/main.py [hunk 3]
```diff
@@ -1 +1,5 @@
-#  write your code here
+from app.animal import Animal, Cat, Dog
```
Comment (ArsenPidhoretskyi) [inline app/main.py:1]:
```suggestion
from app.animal import Animal```

File: app/main.py (hunk 2 above)
Comment (ArsenPidhoretskyi) [inline app/main.py:4]:
```suggestion
def feed_animals(ls_animals: list[Animal]) -> int:
```

File: app/main.py [hunk 4]
```diff
@@ -1 +1,5 @@
...
+
+
+def feed_animals(ls_animals: list[Animal | Cat | Dog]) -> int:
+    return sum([animal.feed() for animal in ls_animals])
```
Comment (ArsenPidhoretskyi) [inline app/main.py:5]:
```suggestion
    return sum(animal.feed() for animal in ls_animals)
```
Converting to list is not needed

**Review by ArsenPidhoretskyi — CHANGES_REQUESTED** [review]

File: app/main.py (hunk 1 above)
[the remark this answers is not in the document]
Comment (goldenuni) [reply app/main.py:4]: image

[the remark this answers is not in the document]
Comment (goldenuni) [reply app/main.py:4]: it is happen, when I don`t import those classes

File: app/main.py (hunk 3 above)
Comment (goldenuni) [reply app/main.py:1]: @ArsenPidhoretskyi 
image

**Review by Bohdan-Salamakha — APPROVED** [review]

**Pull request closed without merging**